storage/src/journal/fixed.rs

```diff
@@ -383,17 +403,18 @@ impl<B: Blob, E: Storage<B>, A: Array> Journal<B, E, A> {
         panic!("no blobs found");
     }
 
-    /// Return the blob containing the oldest unpruned items and its index.
+    /// Return the blob containing the oldest remembered items and its index.
     fn oldest_blob(&self) -> (u64, &B) {
         if let Some((index, blob)) = self.blobs.first_key_value() {
             return (*index, blob);
         }
         panic!("no blobs found");
     }
 
-    /// Allow the journal to prune items older than `min_item_position`. The journal may still
-    /// retain some of these items, for example if they are part of the most recent blob.
-    pub async fn prune(&mut self, min_item_position: u64) -> Result<(), Error> {
+    /// Allow the journal to prune items older than `min_item_position`. The journal may not prune
+    /// all such items in order to respect blob boundaries and maintain its implementation
+    /// invariants.
+    pub async fn forget(&mut self, min_item_position: u64) -> Result<(), Error> {
```

Comment:
Does it make sense to align the naming with `journal::variable` (https://github.com/commonwarexyz/monorepo/blob/0d7677e31b9ac55860f3c8a2ac086934ee8ff78d/storage/src/journal/variable.rs#L544-L572) and `archive` (https://github.com/commonwarexyz/monorepo/blob/0d7677e31b9ac55860f3c8a2ac086934ee8ff78d/storage/src/archive/storage.rs#L498-L503)?

Or is the specific delineation that "forget" may not actually "prune" to the requested position?

Reply:
I think it does align more or less with variable (which is pruning sections vs forgetting items).

the distinction I'm thinking of is pruning is "get rid of this one thing" vs forgetting is "get rid of everything up to this point in time".

Reply:
oh hmm, I guess for archive it is "forgetting items" though.

Reply:
ok it's all "pruning" now.